custom_components/tahoma/sensor.py

```diff
@@ -130,6 +143,18 @@ def update(self):
                 )
             )
 
+        if CORE_ELECTRIC_ENERGY_CONSUMPTION_STATE in self.tahoma_device.active_states:
+            self.current_value = (
+                float(
+                    "{:.2f}".format(
+                        self.tahoma_device.active_states.get(
+                            CORE_ELECTRIC_ENERGY_CONSUMPTION_STATE
+                        )
+                    )
+                )
+                / 1000
```

Comment:
Any reason why you chose kW over W (which is the value returned by TaHoma?)

Reply:
Energy is usually given in kWh rather than Wh, which also improves the readability.